portfolio/src/main/webapp/index.html

```diff
@@ -6,15 +6,29 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+  <style>
+      img{
```

Comment:
Use a class name and put this rule in css file

Reply:
I will make that change.